content/nomos/updates/Athens Offsite Notes.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Athens Offsite Notes
+tags:
+  - nomos-updates
+  - athens-all-hands
+date: 2024-04-15
+lastmod: 2024-04-15
+draft: false
+description: Nomos Offsite Notes
+---
+### `Research`
+
+The Nomos team has attended several meetings with different teams and internally. Most of these have been in terms of finding the right solutions for the Coordination layer, including but not limited to:
+
+- Proof aggregation techniques
+- ZK commitment schemes
+- CL Inter-zone communication
+- Bridges
+
+Based on these discussions, we have compiled a preliminary version of the [Coordination Layer Specification](https://www.notion.so/CL-Specification-9a60bd41167d426abb0cd07a01aab905). There are still unknowns in this line of work, but we are of a unified mind that we are on the right track.
+
+### `Engineering`
+
+At Nomos, there are some needs that could be covered by Vac. To be more precise: understanding on what Vac provides that could be helpful, understanding what is required for Vac to start on that support, a plan/pipeline for both teams to interact. Some of the proposals:
+
+DST:
+
+- At the moment, the most prominent project is the analysis of the Nomos mixnet.
+- This will serve as the first project interaction as it is also the highest priority.
+- Ongoing conversations on what is needed are already in place.
+
+QA:
+
+- Nomos node is in high need of a testing plan. But no real bandwidth for it.
+- Vac has specialized people with the experience and the workforce.
+- Cryptarchia and NomosDA would be the focus. But this is gonna be delayed until seen how the mixnet collaboration goes.
+- As part of this **specifications** will be reviewed by the Vac team so they can be improved. Eventually specifications will be forked and served as part of Vac too.
+
+## `Comms`
+
+##### `Key Market Trends / Research
+
+Privacy Concerns:
+
+- Building our case for privacy our protection and viability of the project. We need to define the terms of privacy and create that shield. Privacy is necessary for personal items to be possible, health, business, institutions. 
+
+Network States:
+
+- Long term play. When network states become a thing we need to be positioned. Excitement right now in crypto is harder to get, but we need to be prepared.
```

Comment:
ok, well, there is no guarantee of indicator that Network States will ever be a thing. This is a weak thing to put into a public note.
Please remove all these statements that are internal, just keep things here that we want to make public